llvm/lib/IR/RepoTicket.cpp

```diff
@@ -25,6 +26,14 @@
 
 using namespace llvm;
 
+#define DEBUG_TYPE "prepo-ticket"
```

Comment:
Issue #8 is about removing the remaining misuse of the “ticket” terminology. I know that this file is called RepoTicket.h (although it shouldn't be). We should be wary of adding more misuses…

Reply:
Fixed, changed from `prepo-ticket` to `prepo-digest`.